util/shared/descriptions.test.mjs

```diff
@@ -16,18 +16,16 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/**
- * This module sets up the mock transport layer immediately, instead of letting the SDK wait 500ms
- */
+import { getMarkdown, getAllMarkdownNames } from './descriptions.mjs'
 
- const win = globalThis || window
+test('proving that getMarkdown is untestable', () => {
+  // This function mutates a variable in the function's global scope and performs side-effects on it.
+  // Unless we setup the entire scenario around this, all we're testing is that a feature of
+  // the javascript syntax works. description[name].
+  expect(getMarkdown('foo')).toBeUndefined()
+})
 
- if (!win.__firebolt) {
-     win.__firebolt = {}
- }
- 
- // wires up the mock transport w/out waiting
- win.__firebolt.mockTransportLayer = true
- 
- // sets a flag that mock defaults impls can use to speed things up, e.g. Lifecycle/defaults.js
- win.__firebolt.automation = true
\ No newline at end of file
+test('and by extension, so is getAllMarkdowns', () => {
+  // Yay, we have tested that Object.keys works as described
+  expect(getAllMarkdownNames('blah')).toStrictEqual([])
+})
```

Comment:
## And here are some tests

The presence of these will allow `npm test` to have something to do.